homeassistant/components/mobile_app/config_flow.py

```diff
@@ -28,7 +28,9 @@ async def async_step_user(
             reason="install_app", description_placeholders=placeholders
         )
 
-    async def async_step_registration(self, user_input=None):
+    async def async_step_registration(
+        self, user_input: dict[str, Any]
+    ) -> ConfigFlowResult:
```

Comment:
`user_input` is always provided during the registration process - it is not optionnal, and never `None`.